### DISCUSSION TITLE
A person born in 1975 belongs to which generation?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#430] [Galaxy Trivia] [answered]
**Bonus:** What was the previous generation called?

**Comment by peterbaccaro** [comment]
A person born in 1975 would belong to the **Gen X** generation (born between 1965 until 1976).
The previous generation was called **Baby Boomers** (born between 1946 and 1964).

**Comment by thuongtruong1009** [comment, marked as answer]
A person born in 1975 belongs to the generation known as Generation X, or Gen X for short. Gen X is generally defined as the generation born between the mid-1960s and the early 1980s, although the exact dates can vary depending on the source. Members of Gen X grew up during a time of significant social, economic, and technological change, including the rise of personal computers, the end of the Cold War, and the emergence of new forms of media such as cable television and the internet.
